galeracluster/source/documentation/jails.rst

```diff
@@ -307,7 +307,7 @@ To start the initial cluster node, run the following commands:
 .. code-block:: console
 
    # ezjail-admin console galera-node
-   # service mysql start --wsrep-new-cluster
+   # service mysql start mysqld_bootstrap
```

Comment:
I don't think this is right for FreeBSD. Can you check, failing which, don't change this yet, by filing another issue

Reply:
Returned to the original. I cannot check, I don't the environment.